### DISCUSSION TITLE
NFT的用途

### DISCUSSION THREAD

**Discussion by eliezhao** [rebase-network/hello-world#18] [Ideas]
NFT+DAO来组织社区，进行治理？

**Comment by eliezhao** [comment]
拥有NFT的人可以获取Rebase社区的治理权，并通过算法让NFT的权重可以随着治理人的投票，或者proposal采用率提升价值。

**Reply by luckyyang** [reply]
这个想法不错，后面有机会可以尝试

**Reply by luckyyang** [reply]
proposal 也可以在这里提交和讨论
